### ISSUE TITLE
CAN NOT WAIT

### ISSUE THREAD

**Issue by VirtualGaming1** [Caltinor/Guild_and_Commerce2#1]
hay if you ever want to have it tested on a server with mods im willing helping out with that and i am using 1.16.4 right now and this sound like a great mod to add in when looked at the project as well. let me know.

**Comment by Caltinor** [comment]
Absolutely!  I'm on a short break right now as I finish grad school, but I should be making strides on this project once that is over.  I will certainly reach out when I have playable builds out.

**Comment by VirtualGaming1** [comment]
Hay Caltinor hope school went well this year for you and want to ask is there any way you could just make a temp Guild or party system for now?

**Comment by Caltinor** [comment]
Actually, I might be able to do just that.  I was planing on releasing the mod in alpha stages where each stage would be the addition of different element (land/guild/money/trade).  I was working on land and protections, but it's been frustrating me, so I could switch gears to guild.

**Comment by VirtualGaming1** [comment]
sorry just look and yea that sound good and i know what you mean "frustrating". when it is not working what you are hoping or the way you want. i been there before
